accelerator/core/core.c

```diff
@@ -640,6 +644,30 @@ status_t broadcast_buffered_txn(const ta_core_t* const core) {
   status_t ret = SC_OK;
   int uuid_list_len = 0;
   hash8019_array_p txn_trytes_array = hash8019_array_new();
+  get_trytes_req_t* req;
```

Comment:
Move the declaration of `req` and `res` to Line 706.

Reply:
It could improve code readability.

Reply:
I moved them to Line 671 instead, since there is a do-while loop at Line 671